dashboards/c#/c#.json

```diff
@@ -0,0 +1,888 @@
+{
```

Comment:
We'll need to update the name of this file as well! Something like "dashboards/c-sharp/dashboard.json" should work fine.

Reply:
Thank you Alec, It worked.. 😀